### PR TITLE
Add "Life Saver"

### PR DESCRIPTION
**Pull request by vpeschenkov**
## Project URL
https://github.com/amiantos/lifesaver

## Category
```screensaver```

## Description
An abstract screensaver based on Conway's Game of Life implemented with SpriteKit

## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -7322,6 +7322,19 @@
...
+      "official_site" : "https://amiantos.net/lifesaver/",
+      "title" : "Life Saver",
+      "screenshots" : [
+        "https://raw.githubusercontent.com/amiantos/lifesaver/master/Design/lifesaver-screenshots.png"
+      ],
+      "short_description" : "An abstract screensaver based on Conway's Game of Life implemented with SpriteKit",
+      "languages" : [
+        "swift"
+      ],
+      "category" : "screensaver"
```
Comment (serhii-londar) [inline applications.json]: @vpeschenkov Now it is array. Please change it to "categories" : [ "screensaver" ]

File: applications.json [hunk 2]
```diff
@@ -7322,6 +7322,21 @@
...
+      "official_site" : "https://amiantos.net/lifesaver/",
+      "title" : "Life Saver",
+      "screenshots" : [
+        "https://raw.githubusercontent.com/amiantos/lifesaver/master/Design/lifesaver-screenshots.png"
+      ],
+      "short_description" : "An abstract screensaver based on Conway's Game of Life implemented with SpriteKit",
+      "languages" : [
+        "swift"
+      ],
+      "category" : [
```
Comment (serhii-londar) [inline applications.json]: "categories" not "category".

Comment (vpeschenkov) [reply applications.json]: @serhii-londar ready

**Comment by serhii-londar** [comment]
@vpeschenkov Thank you 👍

**Pull request merged**